apps/emqx_bridge_dynamo/src/emqx_bridge_dynamo.erl

```diff
@@ -1,4 +1,4 @@
-%%--------------------------------------------------------------------
+%%---------------------------------------------------------------
```

Comment:
Nano nit: revert?

Reply:
Reverted in first commit and force pushed